games-action/omnispeak/omnispeak_keen4-k4.1.4.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Omnispeak - Commander Keen 4 data"
+DESCRIPTION="Commander Keen 4 data files, released in 1991 as shareware, packaged to be used with the Omnispeak binaries."
+HOMEPAGE="https://legacy.3drealms.com/keen4/"
+COPYRIGHT="1991 Id Software, Apogee Inc."
+#No Shareware license?
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI="https://davidgow.net/keen/4keen14.zip"
+CHECKSUM_SHA256="f92e2d02d3779a2e717a50287ef3ed116c22fdf7bcc12ccf676a412e97195cfe"
+SOURCE_DIR=""
+ADDITIONAL_FILES="
+	omni-keen4.rdef.in
+	omni-keen4.sh
+	"
+
+#should be ARCHITECTURES="any" and no secondary eventually
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURE="x86"
+DISABLE_SOURCE_PACKAGE="yes"
+
+PROVIDES="
+	omnispeak_keen4$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
```

Comment:
I would use "SUPPLIMENTS" (*) (mind a typo) here, either way both will be installed.